test/e2e/common.go

```diff
@@ -39,6 +40,24 @@ func LogFromFile(logFile string) {
 	Logf(string(data))
 }
 
+func logTable(title string, columnNames []string, rows [][]string) {
```

Comment:
Why are columnNames separate from rows? We are anyway printing everything as one big table. I would suggest just skipping columnNames for simplicity. Speaking about simplicity, I would also suggest skipping the title. It can be printed separately by the caller and then this function becomes minimal and nice, just doing one thing and doing it well. :slightly_smiling_face:

Reply:
Well I had the same thoughts but as conceptually the names can be formatted  separately like we add a line between them and the other rows we can print then in bold .. I keep them separate for any future nice formatting needed the title also play a role of a comment when calling the function it say what is it about I think it is still simple and nice

Reply:
Ok the formatting I understand. The title I think is a bit strange. It is not a table. The caller can just print it before calling this and then this whole function becomes better focused on one thing.

Reply:
@mboukhalfa any comment? :point_up:

Reply:
I removed column names from the parameter to be the first row

Reply:
@mboukhalfa how does it look like now, is it the same as in PR description?

Reply:
still the same you check on the logs

Reply:
```
  Listing Nodes
  Name:                     Status:    Version:  
  test1-77fdcff7b8-2bfb4    Ready      v1.25.2   
  test1-txbnn               Ready      v1.25.2
```

Reply:
Okay, I though it is changed now based on the above discussion, fine by me